libs/ui/molecules/src/lib/LoginFormControl/LoginFormControl.tsx

```diff
@@ -1,12 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useMutation } from '@tanstack/react-query';
 
 import { Button, FormControl, Input } from '@inheartive/ui/atoms';
+import { apiRoutes } from '@inheartive/data';
+import { NativeSyntheticEvent, TextInputChangeEventData } from 'react-native';
+import { useNavigate } from 'react-router-native';
+import { RoutingPath } from '../../../../../../apps/mobile/src/app/routing/routing-path';
+
+export const storage = {
+  access_token: '',
```

Comment:
This solution is temporary, I did not figure out yet how to store access tokens in react native. If you know how to handle it please let me know

Reply:
Here are a few options:

AsyncStorage: This is a built-in storage system for React Native that allows you to store key-value pairs. You can use AsyncStorage to store the access token and retrieve it later when needed.

SecureStore: This is a library for securely storing sensitive data in React Native. It uses the native Keychain on iOS and the KeyStore on Android, which are both secure storage systems provided by the operating system.

Redux: If you're using Redux in your React Native app, you can store the access token in the Redux store. This makes it easy to access the token throughout the app, but it also means that the token is stored in memory, which could be a security concern if the app is running on a rooted or jailbroken device.

React context: You can use the React context API to store the access token. This allows you to share data between components without having to pass props down manually through multiple levels of components.

It's important to consider the security requirements of your app when choosing a storage method. If your app handles sensitive data, you should use a secure storage system like SecureStore to protect the access token.

Reply:
@KonradKarlinski thanks, will stick with most secure option